core/storage/voiceover/gae_models.py

```diff
@@ -45,16 +45,12 @@ class VoiceoverDict(TypedDict):
     duration_secs: float
```

Comment:
The state domain is not defined in the mypy_imports file. Are we allowed to add state domain in mypy_imports file?
Also, the reason I have created another dict even though one already exists, is to prevent imports from domain to model.

Reply:
Yes, only relevant files will be updated not the mypy_imports file.